README.md

```diff
@@ -1,2 +1,23 @@
-# frontend-developer-challenge
-Simpleweb frontend developer challenge
+# Simpleweb frontend developer challenge
+Using the framework of your choice, please create a frontend application that lists car parks in the UK.
+If you have previously completed the [backend developer challenge](https://github.com/simpleweb/backend-developer-challenge), feel free to use that API, otherwise we will provide you with a URL and authentication details to get the data.
```

Comment:
> we will provide you with a URL and authentication details to get the data.

This actually makes more sense than my original idea or allowing someone to spin up a Docker contain via docker-compose. We could just host it on a free Heroku dyno and include the link within this repo.